### ISSUE TITLE
Data is lost saving a big menu

### ISSUE THREAD

**Issue by mauserrifle** [rainlab/pages-plugin#422]
- OctoberCMS Build: 458
- RainLab Pages Plugin Version: 1.3.1
- PHP Version: 7.3

### Description:

When saving a big menu there seems to be a limit on data. Raw post data is missing menuItem data (not related to post_max_size).

### Steps To Reproduce:

Using the big yaml data below this issue. When saving the menu, data is lost. This causes the last menu item to only have "title". It is impossible to add new items too. Might be a javascript issue.

After saving, my last item looks like:

```
            [7] => Array
                (
                    [title] => Klantenservice
                )
```
Tested using `print_r` in `controllers/Index.php`:

```
    protected function fillObjectFromPost($type)
    {
       .......
        $postData = post();
        print_r($postData);
        die();
        .......
```

Full menu:

```
name: Hoofdmenu
items:
    -
        title: Kilometerverzekering
        nesting: '0'
        type: static-page
        url: ''
        code: ''
        reference: kilometerverzekering
        cmsPage: ''
        replace: '0'
        viewBag:
            isHidden: '0'
            cssClass: ''
            isExternal: '0'
        items:
            -
                title: Kilometerverzekering
                nesting: '0'
                type: static-page
                url: ''
                code: ''
                reference: kilometerverzekering
                cmsPage: ''
                replace: '0'
                viewBag:
                    isHidden: '0'
                    cssClass: ''
                    isExternal: '0'
            -
                title: Vergelijken
                nesting: '0'
                type: static-page
                url: ''
                code: ''
                reference: kilometerverzekering-vergelijken
                cmsPage: ''
                replace: '0'
                viewBag:
                    isHidden: '0'
                    cssClass: ''
                    isExternal: '0'
            -
                title: Afsluiten
                nesting: '0'
                type: static-page
                url: ''
                code: ''
                reference: kilometerverzekering-afsluiten
                cmsPage: ''
                replace: '0'
                viewBag:
                    isHidden: '0'
                    cssClass: ''
                    isExternal: '0'
            -
                title: 'WA Kilometerverzekering'
                nesting: '0'
                type: static-page
                url: ''
                code: ''
                reference: kilometerverzekering-dekking-wa
                cmsPage: ''
                replace: '0'
                viewBag:
                    isHidden: '0'
                    cssClass: ''
                    isExternal: '0'
            -
                title: 'WA + Extra Kilometerverzekering'
                nesting: '0'
                type: static-page
                url: ''
                code: ''
                reference: kilometerverzekering-dekking-wa-extra
                cmsPage: ''
                replace: '0'
                viewBag:
                    isHidden: '0'
                    cssClass: ''
                    isExternal: '0'
            -
                title: 'WA + Casco Kilometerverzekering'
                nesting: '0'
                type: static-page
                url: ''
                code: ''
                reference: kilometerverzekering-dekking-wa-casco
                cmsPage: ''
                replace: '0'
                viewBag:
                    isHidden: '0'
                    cssClass: ''
                    isExternal: '0'
            -
                title: 'Rechtsbijstand Kilometerverzekering'
                nesting: '0'
                type: static-page
                url: ''
                code: ''
                reference: kilometerverzekering-dekking-rechtsbijstand
                cmsPage: ''
                replace: '0'
                viewBag:
                    isHidden: '0'
                    cssClass: ''
                    isExternal: '0'
            -
                title: 'Inzittenden Kilometerverzekering'
                nesting: '0'
                type: static-page
                url: ''
                code: ''
                reference: kilometerverzekering-dekking-schadeverzekering-voor-inzittenden
                cmsPage: ''
                replace: '0'
                viewBag:
                    isHidden: '0'
                    cssClass: ''
                    isExternal: '0'
            -
                title: 'Pechhulp Kilometerverzekering'
                nesting: '0'
                type: static-page
                url: ''
                code: ''
                reference: kilometerverzekering-dekking-pechhulp
                cmsPage: ''
                replace: '0'
                viewBag:
                    isHidden: '0'
                    cssClass: ''
                    isExternal: '0'
            -
                title: 'Polisvoorwaarden Kilometerverzekering'
                nesting: '0'
                type: static-page
                url: ''
                code: ''
                reference: kilometerverzekering-polisvoorwaarden
                cmsPage: ''
                replace: '0'
                viewBag:
                    isHidden: '0'
                    cssClass: ''
                    isExternal: '0'
            -
                title: 'Verzekeringskaart Kilometerverzekering'
                nesting: '0'
                type: static-page
                url: ''
                code: ''
                reference: kilometerverzekering-verzekeringskaart
                cmsPage: ''
                replace: '0'
                viewBag:
                    isHidden: '0'
                    cssClass: ''
                    isExternal: '0'
            -
                title: 'Veelgestelde vragen Kilometerverzekering'
                nesting: ''
                type: url
                url: /kilometerverzekering/veelgestelde-vragen
                code: ''
                reference: ''
                cmsPage: ''
                replace: ''
                viewBag:
                    isHidden: '0'
                    cssClass: ''
                    isExternal: '0'
    -
        title: Autoverzekering
        nesting: '0'
        type: static-page
        url: ''
        code: ''
        reference: autoverzekering
        cmsPage: ''
        replace: '0'
        viewBag:
            isHidden: '0'
            cssClass: ''
            isExternal: '0'
        items:
            -
                title: Autoverzekering
                nesting: '0'
                type: static-page
                url: ''
                code: ''
                reference: autoverzekering
                cmsPage: ''
                replace: '0'
                viewBag:
                    isHidden: '0'
                    cssClass: ''
                    isExternal: '0'
            -
                title: Vergelijken
                nesting: '0'
                type: static-page
                url: ''
                code: ''
                reference: autoverzekering-vergelijken
                cmsPage: ''
                replace: '0'
                viewBag:
                    isHidden: '0'
                    cssClass: ''
                    isExternal: '0'
            -
                title: Afsluiten
                nesting: '0'
                type: static-page
                url: ''
                code: ''
                reference: autoverzekering-afsluiten
                cmsPage: ''
                replace: '0'
                viewBag:
                    isHidden: '0'
                    cssClass: ''
                    isExternal: '0'
            -
                title: 'WA autoverzekering'
                nesting: '0'
                type: static-page
                url: ''
                code: ''
                reference: autoverzekering-dekking-wa
                cmsPage: ''
                replace: '0'
                viewBag:
                    isHidden: '0'
                    cssClass: ''
                    isExternal: '0'
            -
                title: 'WA + Extra autoverzekering'
                nesting: '0'
                type: static-page
                url: ''
                code: ''
                reference: autoverzekering-dekking-wa-extra
                cmsPage: ''
                replace: '0'
                viewBag:
                    isHidden: '0'
                    cssClass: ''
                    isExternal: '0'
            -
                title: 'WA + Casco autoverzekering'
                nesting: '0'
                type: static-page
                url: ''
                code: ''
                reference: autoverzekering-dekking-wa-casco
                cmsPage: ''
                replace: '0'
                viewBag:
                    isHidden: '0'
                    cssClass: ''
                    isExternal: '0'
            -
                title: 'Rechtsbijstand autoverzekering'
                nesting: '0'
                type: static-page
                url: ''
                code: ''
                reference: autoverzekering-dekking-rechtsbijstand
                cmsPage: ''
                replace: '0'
                viewBag:
                    isHidden: '0'
                    cssClass: ''
                    isExternal: '0'
            -
                title: 'Schadeverzekering voor inzittenden'
                nesting: '0'
                type: static-page
                url: ''
                code: ''
                reference: autoverzekering-dekking-schadeverzekering-voor-inzittenden
                cmsPage: ''
                replace: '0'
                viewBag:
                    isHidden: '0'
                    cssClass: ''
                    isExternal: '0'
            -
                title: 'Pechhulp bij de autoverzekering'
                nesting: '0'
                type: static-page
                url: ''
                code: ''
                reference: autoverzekering-dekking-pechhulp
                cmsPage: ''
                replace: '0'
                viewBag:
                    isHidden: '0'
                    cssClass: ''
                    isExternal: '0'
            -
                title: 'Veelgestelde vragen autoverzekering'
                nesting: ''
                type: url
                url: /autoverzekering/veelgestelde-vragen-auto
                code: ''
                reference: ''
                cmsPage: ''
                replace: ''
                viewBag:
                    isHidden: '0'
                    cssClass: ''
                    isExternal: '0'
            -
                title: 'VIP Schadeservice'
                nesting: ''
                type: url
                url: /autoverzekering/vip-schadeservice
                code: ''
                reference: ''
                cmsPage: ''
                replace: ''
                viewBag:
                    isHidden: '0'
                    cssClass: ''
                    isExternal: '0'
    -
        title: Verzekeringen
        nesting: '0'
        type: static-page
        url: ''
        code: ''
        reference: verzekeringen
        cmsPage: ''
        replace: '0'
        viewBag:
            isHidden: '0'
            cssClass: ''
            isExternal: '0'
        items:
            -
                title: 'VoordeelPakket (NIEUW)'
                nesting: ''
                type: url
                url: /voordeelpakket
                code: ''
                reference: ''
                cmsPage: ''
                replace: ''
                viewBag:
                    isHidden: '0'
                    cssClass: ''
                    isExternal: '0'
            -
                title: Aansprakelijkheidsverzekering
                nesting: ''
                type: url
                url: /aansprakelijkheidsverzekering
                code: ''
                reference: ''
                cmsPage: ''
                replace: ''
                viewBag:
                    isHidden: '0'
                    cssClass: ''
                    isExternal: '0'
            -
                title: Inboedelverzekering
                nesting: ''
                type: url
                url: /inboedelverzekering
                code: ''
                reference: ''
                cmsPage: ''
                replace: ''
                viewBag:
                    isHidden: '0'
                    cssClass: ''
                    isExternal: '0'
            -
                title: Woonhuisverzekering
                nesting: ''
                type: url
                url: /woonhuisverzekering
                code: ''
                reference: ''
                cmsPage: ''
                replace: ''
                viewBag:
                    isHidden: '0'
                    cssClass: ''
                    isExternal: '0'
            -
                title: Reisverzekering
                nesting: ''
                type: url
                url: /reisverzekering
                code: ''
                reference: ''
                cmsPage: ''
                replace: ''
                viewBag:
                    isHidden: '0'
                    cssClass: ''
                    isExternal: '0'
            -
                title: Rechtsbijstandverzekering
                nesting: ''
                type: url
                url: /rechtsbijstandverzekering
                code: ''
                reference: ''
                cmsPage: ''
                replace: ''
                viewBag:
                    isHidden: '0'
                    cssClass: ''
                    isExternal: '0'
            -
                title: Gezinsongevallenverzekering
                nesting: ''
                type: url
                url: /gezinsongevallenverzekering
                code: ''
                reference: ''
                cmsPage: ''
                replace: ''
                viewBag:
                    isHidden: '0'
                    cssClass: ''
                    isExternal: '0'
            -
                title: Bromfietsverzekering
                nesting: ''
                type: url
                url: /bromfietsverzekering
                code: ''
                reference: ''
                cmsPage: ''
                replace: ''
                viewBag:
                    isHidden: '0'
                    cssClass: ''
                    isExternal: '0'
            -
                title: Autoverzekering
                nesting: ''
                type: url
                url: /autoverzekering
                code: ''
                reference: ''
                cmsPage: ''
                replace: ''
                viewBag:
                    isHidden: '0'
                    cssClass: ''
                    isExternal: '0'
            -
                title: Kilometerverzekering
                nesting: ''
                type: url
                url: /kilometerverzekering
                code: ''
                reference: ''
                cmsPage: ''
                replace: ''
                viewBag:
                    isHidden: '0'
                    cssClass: ''
                    isExternal: '0'
            -
                title: 'Exclusieve auto verzekeren'
                nesting: ''
                type: url
                url: /autoverzekering/exclusieve-auto
                code: ''
                reference: ''
                cmsPage: ''
                replace: ''
                viewBag:
                    isHidden: '0'
                    cssClass: ''
                    isExternal: '0'
            -
                title: Bestelautoverzekering
                nesting: ''
                type: url
                url: /bestelautoverzekering
                code: ''
                reference: ''
                cmsPage: ''
                replace: ''
                viewBag:
                    isHidden: '0'
                    cssClass: ''
                    isExternal: '0'
            -
                title: Motorverzekering
                nesting: ''
                type: url
                url: /motorverzekering
                code: ''
                reference: ''
                cmsPage: ''
                replace: ''
                viewBag:
                    isHidden: '0'
                    cssClass: ''
                    isExternal: '0'
            -
                title: Scooterverzekering
                nesting: ''
                type: url
                url: /scooterverzekering
                code: ''
                reference: ''
                cmsPage: ''
                replace: ''
                viewBag:
                    isHidden: '0'
                    cssClass: ''
                    isExternal: '0'
            -
                title: Brommobielverzekering
                nesting: ''
                type: url
                url: /brommobielverzekering
                code: ''
                reference: ''
                cmsPage: ''
                replace: ''
                viewBag:
                    isHidden: '0'
                    cssClass: ''
                    isExternal: '0'
            -
                title: 'Oldtimer / klassieker'
                nesting: ''
                type: url
                url: /klassiekerverzekering
                code: ''
                reference: ''
                cmsPage: ''
                replace: ''
                viewBag:
                    isHidden: '0'
                    cssClass: ''
                    isExternal: '0'
            -
                title: Youngtimerverzekering
                nesting: ''
                type: url
                url: /youngtimerverzekering
                code: ''
                reference: ''
                cmsPage: ''
                replace: ''
                viewBag:
                    isHidden: '0'
                    cssClass: ''
                    isExternal: '0'
            -
                title: Caravanverzekering
                nesting: ''
                type: url
                url: /caravanverzekering
                code: ''
                reference: ''
                cmsPage: ''
                replace: ''
                viewBag:
                    isHidden: '0'
                    cssClass: ''
                    isExternal: '0'
            -
                title: Camperverzekering
                nesting: ''
                type: url
                url: /camperverzekering
                code: ''
                reference: ''
                cmsPage: ''
                replace: ''
                viewBag:
                    isHidden: '0'
                    cssClass: ''
                    isExternal: '0'
            -
                title: Pakketverzekering
                nesting: ''
                type: url
                url: /pakketverzekering
                code: ''
                reference: ''
                cmsPage: ''
                replace: ''
                viewBag:
                    isHidden: '0'
                    cssClass: ''
                    isExternal: '0'
            -
                title: MMBS-verzekering
                nesting: ''
                type: url
                url: /mmbs
                code: ''
                reference: ''
                cmsPage: ''
                replace: ''
                viewBag:
                    isHidden: '0'
                    cssClass: ''
                    isExternal: '0'
            -
                title: 'Vereende verzekeringen'
                nesting: ''
                type: url
                url: /vereende
                code: ''
                reference: ''
                cmsPage: ''
                replace: ''
                viewBag:
                    isHidden: '0'
                    cssClass: ''
                    isExternal: '0'
            -
                title: 'Premium verzekeringen'
                nesting: ''
                type: url
                url: /turien-en-co/premium-verzekeringen
                code: ''
                reference: ''
                cmsPage: ''
                replace: ''
                viewBag:
                    isHidden: '0'
                    cssClass: ''
                    isExternal: '0'
            -
                title: Energieverzekering
                nesting: ''
                type: url
                url: /energieverzekering
                code: ''
                reference: ''
                cmsPage: ''
                replace: ''
                viewBag:
                    isHidden: '0'
                    cssClass: ''
                    isExternal: '0'
    -
        title: Zakelijk
        nesting: '0'
        type: static-page
        url: ''
        code: ''
        reference: zakelijk
        cmsPage: ''
        replace: '0'
        viewBag:
            isHidden: '0'
            cssClass: ''
            isExternal: '0'
        items:
            -
                title: 'Autoverzekering Zakelijk'
                nesting: ''
                type: url
                url: /zakelijk/autoverzekering
                code: ''
                reference: ''
                cmsPage: ''
                replace: ''
                viewBag:
                    isHidden: '0'
                    cssClass: ''
                    isExternal: '0'
            -
                title: 'Bestelautoverzekering Bedrijf'
                nesting: ''
                type: url
                url: /zakelijk/bestelautoverzekering
                code: ''
                reference: ''
                cmsPage: ''
                replace: ''
                viewBag:
                    isHidden: '0'
                    cssClass: ''
                    isExternal: '0'
            -
                title: Vrachtwagenverzekering
                nesting: ''
                type: url
                url: /zakelijk/vrachtwagenverzekering
                code: ''
                reference: ''
                cmsPage: ''
                replace: ''
                viewBag:
                    isHidden: '0'
                    cssClass: ''
                    isExternal: '0'
            -
                title: 'Aansprakelijkheid bedrijven'
                nesting: ''
                type: url
                url: /zakelijk/aansprakelijkheidsverzekering
                code: ''
                reference: ''
                cmsPage: ''
                replace: ''
                viewBag:
                    isHidden: '0'
                    cssClass: ''
                    isExternal: '0'
            -
                title: Inventarisverzekering
                nesting: ''
                type: url
                url: /zakelijk/inventarisverzekering
                code: ''
                reference: ''
                cmsPage: ''
                replace: ''
                viewBag:
                    isHidden: '0'
                    cssClass: ''
                    isExternal: '0'
            -
                title: Goederenverzekering
                nesting: ''
                type: url
                url: /zakelijk/goederenverzekering
                code: ''
                reference: ''
                cmsPage: ''
                replace: ''
                viewBag:
                    isHidden: '0'
                    cssClass: ''
                    isExternal: '0'
            -
                title: Gebouwenverzekering
                nesting: ''
                type: url
                url: /zakelijk/gebouwenverzekering
                code: ''
                reference: ''
                cmsPage: ''
                replace: ''
                viewBag:
                    isHidden: '0'
                    cssClass: ''
                    isExternal: '0'
            -
                title: Bedrijfsschadeverzekering
                nesting: ''
                type: url
                url: /zakelijk/bedrijfsschadeverzekering
                code: ''
                reference: ''
                cmsPage: ''
                replace: ''
                viewBag:
                    isHidden: '0'
                    cssClass: ''
                    isExternal: '0'
            -
                title: 'Rechtsbijstand bedrijf'
                nesting: ''
                type: url
                url: /zakelijk/rechtsbijstandverzekering
                code: ''
                reference: ''
                cmsPage: ''
                replace: ''
                viewBag:
                    isHidden: '0'
                    cssClass: ''
                    isExternal: '0'
            -
                title: Zakenreisverzekering
                nesting: ''
                type: url
                url: /zakelijk/zakenreisverzekering
                code: ''
                reference: ''
                cmsPage: ''
                replace: ''
                viewBag:
                    isHidden: '0'
                    cssClass: ''
                    isExternal: '0'
            -
                title: 'Bedrijfspakketverzekering %'
                nesting: ''
                type: url
                url: /zakelijk/pakketverzekering
                code: ''
                reference: ''
                cmsPage: ''
                replace: ''
                viewBag:
                    isHidden: '0'
                    cssClass: ''
                    isExternal: '0'
            -
                title: 'Onroerend goed verzekering'
                nesting: ''
                type: url
                url: /zakelijk/onroerend-goed-verzekering
                code: ''
                reference: ''
                cmsPage: ''
                replace: ''
                viewBag:
                    isHidden: '0'
                    cssClass: ''
                    isExternal: '0'
            -
                title: Taxiverzekering
                nesting: ''
                type: url
                url: /zakelijk/taxiverzekering
                code: ''
                reference: ''
                cmsPage: ''
                replace: ''
                viewBag:
                    isHidden: '0'
                    cssClass: ''
                    isExternal: '0'
            -
                title: Wagenparkverzekering
                nesting: ''
                type: url
                url: /zakelijk/wagenparkverzekering
                code: ''
                reference: ''
                cmsPage: ''
                replace: ''
                viewBag:
                    isHidden: '0'
                    cssClass: ''
                    isExternal: '0'
            -
                title: Bezorgscooter
                nesting: ''
                type: url
                url: /zakelijk/bezorgscooter
                code: ''
                reference: ''
                cmsPage: ''
                replace: ''
                viewBag:
                    isHidden: '0'
                    cssClass: ''
                    isExternal: '0'
            -
                title: Bezorgfiets
                type: url
                url: /zakelijk/bezorgfiets
                code: ''
                viewBag:
                    isHidden: '0'
                    cssClass: ''
                    isExternal: '0'
            -
                title: Garageverzekering
                nesting: ''
                type: url
                url: /zakelijk/garageverzekering
                code: ''
                reference: ''
                cmsPage: ''
                replace: ''
                viewBag:
                    isHidden: '0'
                    cssClass: ''
                    isExternal: '0'
            -
                title: 'Groene Plaat verzekering'
                nesting: ''
                type: url
                url: /zakelijk/groene-plaat-verzekering
                code: ''
                reference: ''
                cmsPage: ''
                replace: ''
                viewBag:
                    isHidden: '0'
                    cssClass: ''
                    isExternal: '0'
            -
                title: 'VvE verzekeringspakket'
                nesting: ''
                type: url
                url: /zakelijk/vve-verzekeringspakket
                code: ''
                reference: ''
                cmsPage: ''
                replace: ''
                viewBag:
                    isHidden: '0'
                    cssClass: ''
                    isExternal: '0'
            -
                title: 'Horeca verzekeringen'
                nesting: ''
                type: url
                url: /zakelijk/horeca-verzekeringen
                code: ''
                reference: ''
                cmsPage: ''
                replace: ''
                viewBag:
                    isHidden: '0'
                    cssClass: ''
                    isExternal: '0'
            -
                title: 'Zakelijke energie'
                nesting: ''
                type: url
                url: /zakelijk/zakelijke-energie
                code: ''
                reference: ''
                cmsPage: ''
                replace: ''
                viewBag:
                    isHidden: '0'
                    cssClass: ''
                    isExternal: '0'
    -
        title: Schade
        nesting: ''
        type: header
        url: ''
        code: ''
        reference: ''
        cmsPage: ''
        replace: ''
        viewBag:
            isHidden: '0'
            cssClass: ''
            isExternal: '0'
        items:
            -
                title: 'Schade melden'
                nesting: ''
                type: url
                url: /schade-melden
                code: ''
                reference: ''
                cmsPage: ''
                replace: ''
                viewBag:
                    isHidden: '0'
                    cssClass: ''
                    isExternal: '0'
            -
                title: 'VIP Schadeservice'
                nesting: ''
                type: url
                url: /schade-melden/vip-schadeservice
                code: ''
                reference: ''
                cmsPage: ''
                replace: ''
                viewBag:
                    isHidden: '0'
                    cssClass: ''
                    isExternal: '0'
            -
                title: Verhaalservice
                nesting: ''
                type: url
                url: /schade-melden/verhaalservice
                code: ''
                reference: ''
                cmsPage: ''
                replace: ''
                viewBag:
                    isHidden: '0'
                    cssClass: ''
                    isExternal: '0'
            -
                title: Schadeformulier
                nesting: ''
                type: url
                url: /schade-melden/schadeformulier
                code: ''
                reference: ''
                cmsPage: ''
                replace: ''
                viewBag:
                    isHidden: '0'
                    cssClass: ''
                    isExternal: '0'
            -
                title: 'Veelgestelde vragen schade'
                nesting: ''
                type: url
                url: /schade-melden/veelgestelde-vragen-schade
                code: ''
                reference: ''
                cmsPage: ''
                replace: ''
                viewBag:
                    isHidden: '0'
                    cssClass: ''
                    isExternal: '0'
    -
        title: Energie
        nesting: ''
        type: header
        url: ''
        code: ''
        reference: ''
        cmsPage: ''
        replace: ''
        viewBag:
            isHidden: '0'
            cssClass: ''
            isExternal: '0'
        items:
            -
                title: 'Energie vergelijken'
                nesting: ''
                type: url
                url: /energie-vergelijken
                code: ''
                reference: ''
                cmsPage: ''
                replace: ''
                viewBag:
                    isHidden: '0'
                    cssClass: ''
                    isExternal: '0'
            -
                title: Overstappen
                nesting: ''
                type: url
                url: /energie-vergelijken/overstappen
                code: ''
                reference: ''
                cmsPage: ''
                replace: ''
                viewBag:
                    isHidden: '0'
                    cssClass: ''
                    isExternal: '0'
            -
                title: Energieleveranciers
                nesting: '1'
                type: static-page
                url: ''
                code: ''
                reference: energie-vergelijken-energieleveranciers
                cmsPage: ''
                replace: '0'
                viewBag:
                    isHidden: '0'
                    cssClass: ''
                    isExternal: '0'
            -
                title: Energieverzekering
                nesting: ''
                type: url
                url: /energieverzekering
                code: ''
                reference: ''
                cmsPage: ''
                replace: ''
                viewBag:
                    isHidden: '0'
                    cssClass: ''
                    isExternal: '0'
            -
                title: 'Veelgestelde vragen energie'
                nesting: ''
                type: url
                url: /energie-vergelijken/veelgestelde-vragen-energie
                code: ''
                reference: ''
                cmsPage: ''
                replace: ''
                viewBag:
                    isHidden: '0'
                    cssClass: ''
                    isExternal: '0'
            -
                title: 'Begrippenlijst energie'
                nesting: ''
                type: url
                url: /energie-vergelijken/begrippenlijst
                code: ''
                reference: ''
                cmsPage: ''
                replace: ''
                viewBag:
                    isHidden: '0'
                    cssClass: ''
                    isExternal: '0'
            -
                title: 'Zakelijke energie vergelijken'
                nesting: ''
                type: url
                url: /zakelijk/zakelijke-energie
                code: ''
                reference: ''
                cmsPage: ''
                replace: ''
                viewBag:
                    isHidden: '0'
                    cssClass: ''
                    isExternal: '0'
    -
        title: 'Nieuws & Blogs'
        nesting: ''
        type: header
        url: ''
        code: ''
        reference: ''
        cmsPage: ''
        replace: ''
        viewBag:
            isHidden: '0'
            cssClass: ''
            isExternal: '0'
        items:
            -
                title: Nieuws
                nesting: ''
                type: url
                url: /nieuws
                code: ''
                reference: ''
                cmsPage: ''
                replace: ''
                viewBag:
                    isHidden: '0'
                    cssClass: ''
                    isExternal: '0'
            -
                title: Blogs
                nesting: ''
                type: url
                url: /blogs
                code: ''
                reference: ''
                cmsPage: ''
                replace: ''
                viewBag:
                    isHidden: '0'
                    cssClass: ''
                    isExternal: '0'
            -
                title: Nieuwsbrief
                nesting: ''
                type: url
                url: /nieuwsbrief
                code: ''
                reference: ''
                cmsPage: ''
                replace: ''
                viewBag:
                    isHidden: '0'
                    cssClass: ''
                    isExternal: '0'
    -
        title: Klantenservice
        nesting: '1'
        type: static-page
        url: ''
        code: ''
        reference: klantenservice
        cmsPage: ''
        replace: '0'
        viewBag:
            isHidden: '0'
            cssClass: ''
            isExternal: '0'
```

**Comment by webstylecenter** [comment]
I can confirm this bug, also happening with our websites, both local and in production environments. (PHP 7.3, nginx, latest mysql)

**Comment by webstylecenter** [comment]
Also, it seems that this bug is over 3 years old and still left unfixed. On the octobercms forums someone also mentioned the same problem: https://octobercms.com/forum/post/static-pages-plugin-error-with-savingloading-menu

**Comment by webstylecenter** [comment]
The issue is caused because of the default max_input_vars value in PHP.ini - If you increase this value, the problem is resolved.

As a possible fix I would suggest not submitting the menu data as an form array, but as just one value with a serialised array. That would fix this for OctoberCMS. Thanks!

**Comment by bennothommo** [comment]
@webstylecenter Thanks for confirming this and for your suggestion. We'll investigate it and see if it's doable.

**Comment by mauserrifle** [comment]
Omg how could I overlook this PHP setting. Thanks @webstylecenter !

**Comment by queengab** [comment]
Hi,

i can also confirm this bug. And used same solution of @webstylecenter .
If your local server is well setted, you can use a custom `user.ini` to increase the value, but yes the best solution is the webstylecenter suggestion.

It is important, often i create menu navigation of websites with the menu component of rainlab.pages.

**Comment by neilcarpenter007** [comment]
Also came across this problem and @webstylecenter 's solution worked for me too.

Thanks! 👍

**Comment by LukeTowers** [comment]
@bennothommo can you look into swapping this away from using separate input vars to just a single one?

**Comment by bennothommo** [comment]
All,

Please feel free to test the fix in #439 and let us know if this resolves the issue for you. I have been able to submit the menu in the OP without issue during my testing.
